models/kyberswap/avalanche_c/kyberswap_avalanche_c_sources.yml

```
@@ -1,12 +1,14 @@
 version: 2
 
 sources:
-  - name: kyber_avalanche_c
+  - name: kyber_avalanche_c_trades
```

We need to make source name matching the name in Dune V2 engine.
```suggestion
  - name: kyber_avalanche_c
```